pages/common/ern.md

```diff
@@ -0,0 +1,36 @@
+# ern
+
+> Electrode Native is a mobile platform that integrates React Native components into mobile applications.
+> More information: <https://www.electrode.io/site/native.html>.
```

Comment:
This seems to be the description of the platform.
I suggest mentioning the CLI:
```suggestion
> Electrode Native platform command line client.
> More information: <https://native.electrode.io/reference/index-6>.
```

Reply:
Thank you, that's a great suggestion and will make the modifications.